Doc/c-api/call.rst

```diff
@@ -59,20 +59,20 @@ This bears repeating:
 
 .. versionchanged:: 3.12
 
-   The :const:`Py_TPFLAGS_HAVE_VECTORCALL` flag is now removed from a class
+   The :c:macro:`Py_TPFLAGS_HAVE_VECTORCALL` flag is now removed from a class
    when the class's :py:meth:`~object.__call__` method is reassigned.
    (This internally sets :c:member:`~PyTypeObject.tp_call` only, and thus
    may make it behave differently than the vectorcall function.)
    In earlier Python versions, vectorcall should only be used with
-   :const:`immutable <Py_TPFLAGS_IMMUTABLETYPE>` or static types.
+   :c:macro:`immutable <Py_TPFLAGS_IMMUTABLETYPE>` or static types.
```

Comment:
For example, this is annotated with `.. data::`. Will that result in a Sphinx warning when `:c:macro:` is used? Ditto for other constants in this PR.

Reply:
It is now annotated with `.. c:macro::`.

`:c:macro:` and `.. data::` are not compatible, as well as `:data:` or `:const:` and `.. c:var::`.

Reply:
Thanks, I missed that change.